Packs/AzureStorage/Integrations/AzureStorage/README.md

```diff
@@ -0,0 +1,637 @@
+| AzureStorage.StorageAccount.sku.name | String | The SKU name. Required for account creation; optional for update. | 
+| AzureStorage.StorageAccount.sku.tier | String | The SKU tier. This is based on the SKU name. | 
+| AzureStorage.StorageAccount.tags | Unknown | Resource tags. | 
+| AzureStorage.StorageAccount.type | String | The type of the resource. | 
+
+
+#### Command Example
+```!azure-storage-account-list```
+
+#### Context Example
+```json
+{
+    "AzureStorage": {
+        "StorageAccount": [
+            {
+                "id": "/subscriptions/subsciption_id/resourceGroups/esource_group_name/providers/Microsoft.Storage/storageAccounts/account_name",
+                "kind": "Storage",
+                "location": "eastus",
+                "name": "account_name",
+                "properties": {
+                    "creationTime": "2018-10-22T22:38:38.8180662Z",
+                    "encryption": {
+                        "keySource": "Microsoft.Storage",
+                        "services": {
+                            "blob": {
+                                "enabled": true,
+                                "keyType": "Account",
+                                "lastEnabledTime": "2018-10-22T22:38:38.9742903Z"
+                            },
+                            "file": {
+                                "enabled": true,
+                                "keyType": "Account",
+                                "lastEnabledTime": "2018-10-22T22:38:38.9742903Z"
+                            }
+                        }
+                    },
+                    "minimumTlsVersion": "TLS1_1",
+                    "networkAcls": {
+                        "bypass": "AzureServices",
+                        "defaultAction": "Allow",
+                        "ipRules": [],
+                        "virtualNetworkRules": []
+                    },
+                    "primaryEndpoints": {
+                        "blob": "",
+                        "file": "",
+                        "queue": "",
+                        "table": ""
+                    },
+                    "primaryLocation": "eastus",
+                    "privateEndpointConnections": [],
+                    "provisioningState": "Succeeded",
+                    "secondaryLocation": "westus",
+                    "statusOfPrimary": "available",
+                    "statusOfSecondary": "available",
+                    "supportsHttpsTrafficOnly": false
+                },
+                "sku": {
+                    "name": "Standard_GRS",
+                    "tier": "Standard"
+                },
+                "tags": {
+                    "ms-resource-usage": "azure-cloud-shell"
+                },
+                "type": "Microsoft.Storage/storageAccounts"
+            },
+            {
+                "id": "/subscriptions/subsciption_id/resourceGroups/esource_group_name/providers/Microsoft.Storage/storageAccounts/account_name",
```

Comment:
one output example is good enough, you can delete the rest as it makes the readme cumbersome.

Reply:
done.